### ISSUE TITLE
the  stable of convergence

### ISSUE THREAD

**Issue by alexliyang** [liyunsheng13/micronet#9]
I trained about 300 epoches, when I use the model to test , different epoches test results are very different ， sometimes the result is good ， sometimes  the result is very bad， how to resolve this problem

**Comment by liyunsheng13** [comment]
Yes, it is not that stable during training especially for the small model and the first hundreds of epochs. However, it will become more stable in the last 100 epochs when the learning rate is decayed to be very small.
